templates/advancement/item-choice-config-levels.hbs

```diff
@@ -0,0 +1,31 @@
+<fieldset class="card levels">
+    <legend>{{ localize "DND5E.ADVANCEMENT.ItemChoice.FIELDS.choices.label" }}</legend>
+    <div class="header">
+        <span class="level-header">
+            {{ localize "DND5E.AbbreviationLevel" }}
+        </span>
+        <span id="{{ partId }}-choices" class="choices-header">
```

Comment:
Seems weird to give `<span>`s an `id`. If we're just doing this for the sake of labelling the `<input>`s, can we put an `aria-label` on them directly? Or is there some other benefit to using `aria-labelledby` here?